<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverWidgetClass</class>
 <widget class="QMainWindow" name="SolverWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TubeOracle-Solver</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="cenlayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QGroupBox" name="PreParaBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>991</width>
      <height>659</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Config Solver Parameters</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>351</width>
       <height>611</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QWidget" name="loadpage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>349</width>
           <height>505</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="label">
          <string>加载模型</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>331</width>
            <height>491</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="loadlayout" stretch="1,1,13">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>加载网格模型：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,8">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="loadbtn">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>加载</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pathedit"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>291</width>
                <height>381</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,3,1">
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>10</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>10</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="entityNum"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="font">
                      <font>
                       <family>Arial</family>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>实体数量：</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="eleNum"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <family>Arial</family>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>三角单元数：</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="nodeNum"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <family>Arial</family>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>节点数：</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="met">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="title">
                  <string>材料</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <widget class="QWidget" name="verticalLayoutWidget">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>20</y>
                    <width>241</width>
                    <height>181</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,3">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <property name="leftMargin">
                    <number>10</number>
                   </property>
                   <property name="topMargin">
                    <number>10</number>
                   </property>
                   <property name="rightMargin">
                    <number>10</number>
                   </property>
                   <property name="bottomMargin">
                    <number>10</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="sizeConstraint">
                      <enum>QLayout::SetDefaultConstraint</enum>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>弹性模量(GPa)：</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="eedit">
                       <property name="text">
                        <string>18</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>泊松比：</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="vedit">
                       <property name="text">
                        <string>0.25</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="initflag">
                     <property name="text">
                      <string>是否初始化</string>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="init">
                 <property name="text">
                  <string>初始化有限元环境</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="boundarypage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>349</width>
           <height>505</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <attribute name="label">
          <string>边界条件</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>341</width>
            <height>461</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="bonlayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="treeWidget">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>边界设置</string>
              </property>
             </column>
             <item>
              <property name="text">
               <string>力边界</string>
              </property>
              <item>
               <property name="text">
                <string>集中力</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>分布力</string>
               </property>
              </item>
             </item>
             <item>
              <property name="text">
               <string>位移边界</string>
              </property>
              <item>
               <property name="text">
                <string>水平位移</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>竖直位移</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>固定约束</string>
               </property>
              </item>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="computepage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>349</width>
           <height>505</height>
          </rect>
         </property>
         <attribute name="label">
          <string>求解</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>331</width>
            <height>461</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="sovlayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_6">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>291</width>
                <height>447</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="layoutxxxx" stretch="1,1,1">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="leftMargin">
                <number>10</number>
               </property>
               <property name="topMargin">
                <number>20</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>20</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,2">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <property name="leftMargin">
                  <number>10</number>
                 </property>
                 <property name="topMargin">
                  <number>10</number>
                 </property>
                 <property name="rightMargin">
                  <number>20</number>
                 </property>
                 <property name="bottomMargin">
                  <number>280</number>
                 </property>
                 <item>
                  <widget class="QComboBox" name="algorithm">
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <item>
                    <property name="text">
                     <string>高斯消去法</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>求解算法：</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QRadioButton" name="solveflag">
                 <property name="text">
                  <string>已求解</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="compute">
                 <property name="text">
                  <string>求解</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="postpage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>349</width>
           <height>505</height>
          </rect>
         </property>
         <attribute name="label">
          <string>后处理</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>341</width>
            <height>451</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="posylayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_8">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>321</width>
                <height>161</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="plot">
                 <property name="text">
                  <string>绘制云图</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="outport">
                 <property name="text">
                  <string>数据导出</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="resetview">
   <property name="text">
    <string>重置视图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
